breathecode/registry/actions.py

```diff
@@ -261,6 +259,19 @@ def set_blob_content(repo, path_name, content, file_name, branch='main'):
     return repo.update_file(file[0].path, f'Update {file_name}', content, file[0].sha)
 
 
+def generate_screenshot(url: str, dimension: str = '1200x630', **kwargs):
+    screenshot_key = os.getenv('SCREENSHOT_MACHINE_KEY', '')
+    params = {
+        'key': screenshot_key,
+        'url': url,
+        'dimension': dimension,
+        **kwargs,
+    }
+    request = requests.request('GET', 'https://api.screenshotmachine.com', params=params, timeout=8)
```

Comment:
should be fine build a async version called `agenerate_screenshot` due to the performance of sync endpoint is degraded considerably while it's waiting, use aiohttp instead